GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/BasicFileSystemTests.cs

```diff
@@ -941,7 +935,38 @@ public void RunPythonExecutable()
             result.Output.ShouldContain("3.14");
 
             GitProcess.Invoke(this.Enlistment.RepoRoot, "checkout " + this.Enlistment.Commitish);
-       }
+        }
+
+        private void CheckForDeletedFile(string filePath, FileSystemRunner fileSystem)

```

Comment:
Naming suggestion: When reading the above tests it wasn't clear what `CheckForDeletedFile` would be checking.

Maybe rename this to something like `ConfirmDeletedWhileOpen`?

Reply:
How about `VerifyExistenceAfterDeleteWhileOpen`?

Reply:
> How about `VerifyExistenceAfterDeleteWhileOpen`?

Even better! 😃